hydra-explorer/json-schemas/hydra-explorer-api.yaml

```
@@ -0,0 +1,206 @@
+openapi: 3.0.0
+info:
+  title: Head Explorer API
+  version: 1.0.0
```

Could: use latest hydra version (as long as the explorer is in the same repo / tagged together with `hydra-node`)

```suggestion
  version: 0.16.0
```